cmd/api-errors.go

```diff
@@ -955,8 +955,6 @@ func toAPIErrorCode(err error) (apiErr APIErrorCode) {
 		apiErr = ErrEntityTooSmall
 	case NotImplemented:
 		apiErr = ErrNotImplemented
-	case PolicyNotFound:
-		apiErr = ErrNoSuchBucketPolicy
```

Comment:
any reason to remove this? who is returning this error?

Reply:
Previously it was two error types for the same error `BucketPolicyNotFound` and `PolicyNotFound`.  Now `PolicyNotFound` is removed.